app/views/pwa/manifest.json.jb

```diff
@@ -0,0 +1,30 @@
+manifest = {
+  name: "Kaigi on Rails 2023 conference-app",
+  short_name: "Kaigi on Rails 2023",
+  theme_color: "#84240f",
+  background_color: "#fff",
+  display: "standalone",
+  orientation: "portrait",
+  scope: "/",
+  start_url: "/",
+  icons: [
+    {
+      src: image_path("icons/2023/512.png"),
+      type: "image/png",
+      sizes: "512x512",
+    },
+    {
+      src: image_path("icons/2023/1024.png"),
+      type: "image/png",
+      sizes: "1024x1024",
+    },
+    {
+      src: image_path("icons/2023/1024_maskable.png"),
+      type: "image/png",
+      sizes: "1024x1024",
+      purpose: "maskable",
+    },
+  ],
+}
+
+manifest
```

Comment:
Do we need this line instead of just returning a literal?

Reply:
tashikani

Reply:
9d3d3cf